code/modules/reagents/reagent_containers/glass/beaker.dm

```diff
@@ -40,6 +40,12 @@
 		var/mutable_appearance/label = mutable_appearance(icon, label_icon)
 		add_overlay(label)
 
+/obj/item/reagent_containers/glass/beaker/cryoxadone
+	preloaded_reagents = list("cryoxadone" = 30)
+	spawn_blacklisted = TRUE
+
+/obj/item/reagent_containers/glass/beaker/sulphuric
+	preloaded_reagents = list("sacid" = 60)
```

Comment:
This probably was moved up here by accident, subtypes belong under `//// Subtypes ////` comment, I believe.

Reply:
And there are a lot of "objects with only changed contents" and they are all "somewhere right under their parent", but as you may see, they are also under `//// Subtypes ////` section, with actually distinct beaker types being on top, with "reagent subtypes" being under those.
I don't see why cryoxadone and sacid beakers should be an exception.

Reply:
Agreed.

Reply:
Made requested changes.